packages/aquedux-server/src/network/routes/fragment.js

```diff
@@ -1,12 +1,14 @@
+import configManager from '../../managers/configManager'
 import { selectors } from '../../reducers'
 
+const { queueLimit } = configManager.getConfig()
+
 export default (store, route) => (req, res) => {
   const channelName = req.url.substr((route + '/fragments/').length)
   try {
     res.writeHead(200, { 'Content-Type': 'application/json' })
-    const fragmentIndex = Math.floor(
-      selectors.queue.getCursor(store.getState(), channelName) / selectors.queue.getQueueLimit(store.getState())
-    )
+    const cursor = selectors.queue.getCursor(store.getState(), channelName)
+    const fragmentIndex = queueLimit === 0 ? cursor : Math.floor(cursor / queueLimit)
```

Comment:
The fragment index should return 0 if queueLimit === 0.

Reply:
Because in case of an infinite sized queue, there would be 1 infinite sized fragment.